templates/components/products/product-view.html

```diff
@@ -274,7 +274,14 @@ <h2 class="productView-brand">
     </article>
 </div>
 
-<div id="previewModal" class="modal modal--large" data-reveal>
+<div
+    id="previewModal"
+    class="modal modal--large"
+    role="alertdialog"
```

Comment:
`alertdialog` role should only used when an alert, error, or warning occurs. Can we change it to `dialog`?

Reply:
@yurytut1993 above, I've added three videos and comments for them